src/components/widgets/MetadataWidget/BreadcrumbWidget/BreadcrumbWidget.tsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
-import { EuiBadge } from "@elastic/eui";
+import {EuiBadge, EuiFlexItem, EuiLoadingSpinner, EuiText} from "@elastic/eui";
 import {OlsApi} from "../../../../api/OlsApi";
 import {useQuery} from "react-query";
+import {getPreferredOntologyJSON} from "../index";
 
 export interface BreadcrumbWidgetProps {
-  iri?: string;
+  iri: string;
   ontologyId?: string;
   api: string;
   /**
```

Comment:
Not important for the changes but nevertheless, this is the description for the parameter parameter and should be on line 29.

Reply:
Done!